modules/ding_react/ding_react.module

```diff
@@ -321,9 +322,10 @@ function ding_react_follow_searches_url() {
   return variable_get('ding_react_follow_searches_url', DING_REACT_FOLLOW_SEARCHES_STAGE_URL);
 }
 
-function ding_react_admin_settings_form() {
-  $form = [];
-
+/**
+ * Admin settings for the module.
+ */
+function ding_react_admin_settings_form($form, &$form_state) {
```

Comment:
Normally we have places these admin forms in their own files (`ding_react.admin.inc`) to keep the module file code more clean. Why the change of pattern here?